src/renderer/utils/ModData.ts

```diff
@@ -0,0 +1,37 @@
+import { Mod, ModTrack, RollingReleaseModel } from "renderer/utils/InstallerConfiguration";
+import { GitVersions } from "@flybywiresim/api-client";
+
+export type ReleaseInfo = {
+    name: string,
+    releaseDate: Date,
+    changelogUrl?: string,
+}
+
+export class ModData {
```

Comment:
Should be called `AddonData`. We want to get rid of the mod term.

Reply:
Fixed